typed-racket-lib/typed-racket/base-env/base-types-extra.rkt

```diff
@@ -17,7 +17,7 @@
 (define-other-types
   -> ->* case-> U Rec All Opaque Vector
   Parameterof List List* Class Object Values Instance Refinement
-  pred Struct Struct-Type Prefab Top Bot)
+  pred Struct Struct-Type Prefab Top Bot Distinction)
```

Comment:
Ok I just fixed it. There was a missing call to resolve in there, and that fixed it. I also added a version of that example as another test.